firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1161,6 +1161,10 @@ class BrowserCoordinator: BaseCoordinator,
         router.push(webviewViewController)
     }
 
+    func popToBVC() {
+        router.popToViewController(browserViewController, reason: .deeplink, animated: true)
```

Comment:
The default is `true` I think we can remove this?
```suggestion
        router.popToViewController(browserViewController, reason: .deeplink)
```